lib/main.dart

```diff
@@ -1,67 +1,315 @@
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
-
+import 'package:flutter/services.dart';
 void main() {
-  runApp(const MyApp());
+  WidgetsFlutterBinding.ensureInitialized();
+  SystemChrome.setPreferredOrientations([DeviceOrientation.landscapeLeft])
```

Comment:
Your app is currently always displaying in landscape mode (even if your device is currently in portait mode). Ideally it should handle both of them (even if the layout won't be perfect)

Reply:
Thanks, it was a bit of a "crutch" to make a software landscape orientation, my emulator had a problem with the change of orientation, but I'm fixing it :)